public/docs/ts/latest/guide/_data.json

```diff
@@ -28,29 +28,32 @@
     "title": "Dependency Injection",
     "intro": "Angular's dependency injection system creates and delivers dependent services \"just-in-time\"."
   },
- 
+
   "template-syntax": {
     "title": "Template Syntax",
     "intro": "How to write templates that display data and consume user events with the help of data binding."
   },
-    
+
   "pipes": {
     "title": "Pipes",
     "intro": "Pipes transform displayed values within a template"
   },
-  
+
   "attribute-directives": {
     "title": "Attribute Directives",
     "intro": "Attribute directives attach behavior to elements."
   },
-  
+
   "hierarchical-dependency-injection": {
     "title": "Hierarchical Injectors",
     "intro": "Angular's hierarchical dependency injection system supports nested injectors in parallel with the component tree."
   },
-  
+  "structural-directives": {
+    "title": "Structural Directives",
+    "intro": "Angular has a powerful template and transclusion engine that lets you easily manipulate the DOM structure of your elements."
```

Comment:
I'd leave out " and transclusion engine". The word is just begging for trouble and doesn't clarify anything here anyway.